backend/server/manage.py

```diff
@@ -0,0 +1,14 @@
+from oceanhub import app
+from oceanhub.scripts.shell import make_shell_context
+from flask_script import Manager, Command, Shell
```

Comment:
Hmm... Interesting. I haven't used flask much, but didn't know about this Manager feature!

Reply:
yeah the manager stuff lets you bind Flask with other outside features

Reply:
Is this similar to Django's `manage.py`?

Reply:
yup, pretty much